packages/frontend/styles/dds-icons.css

```
@@ -0,0 +1,1851 @@
+@charset "UTF-8";
+.dds__icon {
+  width: 1em;
+  height: 1em;
+  fill: currentColor;
+}
+
+@font-face {
+  font-family: "dds-icons";
+  font-style: normal;
+  font-weight: normal;
+  src: url("//dds.dell.com/icons/2.8.0/dds-icons.woff2") format("woff2"), url("//dds.dell.com/icons/2.8.0/dds-icons.woff") format("woff");
+}
+.dds__icon {
+  font-family: "dds-icons" !important;
+}
+.dds__icon {
+  -webkit-font-smoothing: auto;
+  -moz-osx-font-smoothing: auto;
+  font-size: inherit;
+  font-style: normal;
+  font-feature-settings: normal;
+  font-variant: normal;
+  font-weight: normal;
+  text-transform: none;
+  line-height: 1em;
+}
```

We can deduplicate `dds__icon` selector in a simple way:

```suggestion
@font-face {
  font-family: "dds-icons";
  font-style: normal;
  font-weight: normal;
  src: url("//dds.dell.com/icons/2.8.0/dds-icons.woff2") format("woff2"), url("//dds.dell.com/icons/2.8.0/dds-icons.woff") format("woff");
}
.dds__icon {
  font-family: "dds-icons" !important;
  fill: currentColor;
  width: 1em;
  height: 1em;
  -webkit-font-smoothing: auto;
  -moz-osx-font-smoothing: auto;
  font-size: inherit;
  font-style: normal;
  font-feature-settings: normal;
  font-variant: normal;
  font-weight: normal;
  text-transform: none;
  line-height: 1em;
}
```